src/index.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="webpack-env" />
+
+import type { Meta, StoryContext as _SB_StoryContext, StoryObj } from '@storybook/svelte';
+
+import Story from './components/Story.svelte';
+
+export type { Story };
+export { setTemplate } from './components/context.svelte.js';
+
+// FIXME: We don't use webpack anymore(?)
+if (module?.hot?.decline) {
+  module.hot.decline();
+}
+
+// TODO: Consult if these are correct types
```

Comment:
And lastly, I'd like to consult if I've picked correct types from `@storybookjs/types` & `@storybookjs/svelte`.

Both this addon type helper `Args` and `StoryContext` are used for writing "templates" _(via Svelte snippets)_. I have low confidence on whether I should've used `Meta<"args">` for the `StoryContext` _(from `@storybookjs/svelte`)_, which requires `StrictArgs`. I've got no idea which one are they.